example/data.js

```diff
@@ -73,6 +73,22 @@ const bumpSyncToken = function(cal) {
   cal.syncToken = parts.slice(0, -1).join('/') + '/' + (parseInt(parts[parts.length - 1]) + 1);
 };
 
+const eventFilter = (start, end, eventStart, eventEnd) =>
```

Comment:
Please match the coding style of this repository and only use arrow functions for anonymous callback functions.

Reply:
Should this be documented in the `README` under `Contributing`. Or is this covered by eslint?